tasks/basic-sanity-check-task.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: basic-sanity-check
+spec:
+  stepTemplate:
+    volumeMounts:
+      - name: project
+        mountPath: /workspace/source/image
+  params:
+    - name: sourceImage
+      description: the fully qualified image name
+      default: quay.io/cvpops/test-index:v4.10
+  steps:
+    - name: clone-git-repo
+      image: alpine/git
```

Comment:
using dockerhub could be problematic do to pull limitations -> not possible to run in the office.

Reply:
Got rid of it, no use of cloning at all.